src/main/java/gov/usgs/wma/waterdata/openapi/ConfigOpenApi.java

```diff
@@ -10,7 +10,7 @@
 import io.swagger.v3.oas.models.servers.Server;
 
 @Configuration
-public class OpenApiConfig {
+public class ConfigOpenApi {
```

Comment:
Lesson learned on Pubs, OpneAPIConfig is a bean deep in the auto configuration and having our own class with the same name causes issues.